src/api/controllers/user-controllers.ts

```diff
@@ -64,4 +64,19 @@ const updateUserProfile = async (req: Request, res: Response, next: NextFunction
         next(err);
     }
 };
-export { getAllUsers, createUser, getUserById, getUserByEmail, updateUserProfile };
+
+const addNewUserActivity = async (req: Request, res: Response, next: NextFunction) => {
+    const { id } = req.params;
+    const { activity, address = null } = req.body;
+    try {
+        /** Call to service layer */
+        const result = await userServices.addNewUserActivity(id, activity, address);
```

Comment:
Where are you storing the images coming with the request?
Since images are fairly large, the endpoint should support multi-part files.
I think we can use firebase storage for storing images